src/Hooks/Database/resources/config/database.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the tenancy/tenancy package.
+ *
+ * Copyright Laravel Tenancy & Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://tenancy.dev
+ * @see https://github.com/tenancy
+ */
+
+return [
+    /*
+     * Automatic tenant database handling.
+     */
+
+    'auto-create' => env('TENANCY_DATABASE_AUTO_CREATE', true),
+
+    'auto-update' => env('TENANCY_DATABASE_AUTO_UPDATE', true),
+
+    'auto-delete' => env('TENANCY_DATABASE_AUTO_DELETE', true),
+];
```

Comment:
Not another configuration file? Can't we use a static setter instead?

Reply:
Do we really want to put in the effort in avoiding a config file for something so small?